ts/screens/profile/FiscalCodeScreen.tsx

```diff
@@ -113,10 +113,8 @@ class FiscalCodeScreen extends React.PureComponent<Props, State> {
       const maybeCodiceCatastale = CodiceCatastale.decode(
         this.props.profile.fiscal_code.substring(11, 15)
       );
-      // if municipality data are none we request a load
-      if (pot.isNone(this.props.municipality.data)) {
-        maybeCodiceCatastale.map(c => this.props.loadMunicipality(c));
-      }
+
+      maybeCodiceCatastale.map(c => this.props.loadMunicipality(c));
```

Comment:
this PR lgtm
btw it raises up a bad behavior we have to fix. Here the way to reproduce:

- open the FiscalCode
- the municipality will be loaded and shown
- go back
- shutdown the local server (simulate no connection)
- go again into the FiscalCode
- **the municipality data is empty**

we should show the last stored value.
To do so you have to change the municipality reducer (it's up to you find where it is :D)
In particular you have to change the state returned in front of loading request: now it returns an empty loading state. We have to return `someLoading` if there is a previous value into the store, otherwise `noneLoading` as it is